doc/content/source/bcs/SchottkyEmissionBC.md

```diff
@@ -1,20 +1,46 @@
 # SchottkyEmissionBC
 
-!alert construction title=Undocumented Class
-The SchottkyEmissionBC has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /BCs/SchottkyEmissionBC
 
 ## Overview
 
-!! Replace these lines with information regarding the SchottkyEmissionBC object.
+`SchottkyEmissionBC` is an outflow boundary condition that assumes the electron current density is defined by field ehanced thermionic emission, Schottky emission.
+
+Using a Richardson equation for the field emission, the electron current density is defined as
+
+\begin{equation}
+a_{e} =
+\begin{cases}
+1, & \nabla (V) \cdot \textbf{n} > 0\\
+0, & \nabla (V) \cdot \textbf{n} \leq 0\\
+\end{cases} \\[10pt]
+\textbf{J}_{\textbf{e}} \cdot \textbf{n} = A_{G} T^{2} \exp\left( \frac{\text{-} \left( \phi - \Delta \phi \right) }{ k_{B} T} \right) \\[10pt]
+F =  \left( 1-a_{e} \right) \gamma \left( \text{-} \nabla V \right) \cdot \textbf{n} \\[10pt]
+\Delta \phi = \sqrt{\frac{e^{3} F}{4 \pi \epsilon_{0}}} 
+\end{equation}
+
+Where $\textbf{J}_{\textbf{e}}$ is the electron current density, $A_{G}$ is the Richardson coefficient, $T$ is the temperature of the cathode, $k_{B}$ is Boltzmann constant in units of eV/K, $\phi$ is the local work function, $\Delta \phi$ is the difference in the work funtion due to the electric field, $F$ is the local field, $\textbf{n}$ is the normal vector of the boundary, $\gamma$ is the field enhancement factor, and $V$ is the potential. $a_{e}$ is defined such that the outflow is only defined when the drift velocity is direct towards the wall and zero otherwise. With the electron current density, the outward electron flux is defined as
```

Comment:
```suggestion
Where $\textbf{J}_{\textbf{e}}$ is the electron current density, $A_{G}$ is the Richardson coefficient, $T$ is the temperature of the cathode, $k_{B}$ is Boltzmann constant in units of eV/K, $\phi$ is the local work function, $\Delta \phi$ is the difference in the work funtion due to the electric field, $F$ is the local field, $\textbf{n}$ is the normal vector of the boundary, $\gamma$ is the field enhancement factor, and $V$ is the potential. $a_{e}$ is defined such that the outflow is only defined when the drift velocity is directed towards the wall and zero otherwise. With the electron current density, the outward electron flux is defined as
```